docs/admin/access.rst

```diff
@@ -212,35 +231,18 @@ keep managing your permissions site-wide for these groups too, choose an
 appropriate value for the :guilabel:`Project selection` (e.g.
```

Comment:
this makes no sense to me.
"to manage screenshots appropriate value" ?? and then, something.

Reply:
resolved

Reply:
managed